### PR TITLE
break out hasPVs into two separate functions

### PR DESCRIPTION
**Pull request by alaypatel07**

### CODE REVIEW

File: pkg/controller/migmigration/task.go [hunk 1]
```diff
@@ -1296,18 +1296,22 @@ func (t *Task) getPVCs() map[k8sclient.ObjectKey]migapi.PV {
 
 // Get whether the associated plan lists not skipped PVs.
 // First return value is PVs overall, and second is limited to Move or snapshot copy PVs
```
Comment (dymurray) [inline pkg/controller/migmigration/task.go:1298]: Fix comment

**Review by dymurray — APPROVED** [review]

File: pkg/controller/migmigration/task.go [hunk 2]
```diff
@@ -1313,19 +1313,22 @@ func (t *Task) getPVCs() map[k8sclient.ObjectKey]migapi.PV {
...
-		}
 		if pv.Selection.Action != migapi.PvSkipAction {
-			anyPVs = true
+			return true
+		}
+	}
+	return false
+}
+
+func (t *Task) isAnyPVsUsingSnapshotMethod() bool {
```
Comment (sseago) [inline pkg/controller/migmigration/task.go:1325]: This should be renamed to `isAnyPVsUsingMoveOrSnapshotCopyMethod` and the check needs to include `migapi.PvMoveAction ||`

Comment (sseago) [reply pkg/controller/migmigration/task.go:1325]: Also, either change `are` to `is` in the name, or change `PVs` to `PV`.

**Review by sseago — CHANGES_REQUESTED** [review]
Looks good overall, but the refactor lost the check for `move` PVs. We need to include both Move and Snapshot Copy PVs in the stage backup, which means the check as to whether we need a stage backup also needs to include the move PVs.

File: pkg/controller/migmigration/task.go (hunk 2 above)
Comment (alaypatel07) [reply pkg/controller/migmigration/task.go]: +1 good catch, I updated the PR with suggested changes

**Comment by djwhatle** [comment]
@alaypatel07 @dymurray @sseago what's left on this PR?

**Comment by alaypatel07** [comment]
too late to merge this now

**Pull request closed without merging**